packages/api-openfin/message.js

```diff
@@ -0,0 +1,28 @@
+if (!window.ssf) {
+  window.ssf = {};
+}
+
+class MessageService {
+  // Window ID should be in the form 'application-uuid:window-name'
+  static send(windowId, topic, message) {
+    const splitId = windowId.split(':');
```

Comment:
I'm going to make a note of this little trick, I use object destructuring all the time, but forget about arrays!